gnowledge/gstudio
# BaseCourseGroup Implementation

For any course that will be created henceforth, will be an instance of  **Group** and a member of new GST **BaseCourseGroup**.

What to test:
1. After taking pull, run following commands:
   - `python manage.py filldb`
   - This will create new GST BaseCourseGroup
   - `python manage.py create_schema ATs.csv`
   - This will reset start_time and end_time ATs
2. Click on `Explore` from header. Check BaseCourses listing (you should log in with admin account to view the base courses)
3. Create new course. After saving, it should redirect to course-player UI, with only 3 tabs, About, Content and Raw Material.
4. Try doing some editings to the same course and saving it.
5. Add course structure, files in raw material and announce it. This should create a CourseEvent and redirect to course player with 5 tabs, About, Content, Raw material, Gallery, Notebook.

Test whether all of above said actions are executed without any issue.

Also, the existing courses should also work smoothly.
